src/main/java/echoserver/ClientHandler.java

```diff
@@ -7,13 +7,13 @@
 import java.net.ServerSocket;
 import java.net.Socket;
 
-public class EchoServer extends Thread {
+public class ClientHandler extends Thread {
```

Comment:
Think its preferable to implement `Runnable` rather than extend `Thread`

Reply:
Thanks! I implemented this in a later commit